pkg/abi/linux/fuse.go

```diff
@@ -20,6 +20,9 @@ type FUSEOpcode uint32
 // +marshal
 type FUSEOpID uint64
 
+// FUSE_ROOT_ID is the id of root inode.
+const FUSE_ROOT_ID = 1
```

Comment:
Is this used by other packages? Is it a const commonly known in Unix? If not, better to put in fuse package and use go style naming convention

Reply:
it is here https://elixir.bootlin.com/linux/v4.4/source/include/uapi/linux/fuse.h#L143 in Linux, so use C style name should be fine. Feel it's fine in this file since it is defining lots of constants for FUSE.